### PR TITLE
Tfa forms

### PR DESCRIPTION
**Pull request by eatonphil**
screen shot 2017-05-16 at 9 41 46 am
screen shot 2017-05-16 at 9 42 03 am

### CODE REVIEW

File: src/profile/components/TwoFactor.js [hunk 1]
```diff
@@ -14,68 +14,52 @@ export default class TwoFactor extends Component {
...
-      if (!response.json) {
-        // eslint-disable-next-line no-console
-        return console.error(response);
-      }
+  onSubmit = async () => {
+    const { dispatch, tfaEnabled } = this.props;
 
-      const errors = await reduceErrors(response);
-      this.setState({ errors: errors });
+    const requests = [() => toggleTFA(!tfaEnabled)];
```
Comment (na3d) [inline src/profile/components/TwoFactor.js]: Should we have loading set to true before this?

Comment (eatonphil) [inline src/profile/components/TwoFactor.js]: No, that's handled by dispatchOrStoreErrors

Comment (na3d) [inline src/profile/components/TwoFactor.js]: Ah, the state wouldn't change until then, so multiple calls wouldn't be a big deal.

Comment (eatonphil) [inline src/profile/components/TwoFactor.js]: nope it handles all that appropriately when it is called.

**Comment by eatonphil** [comment]
Thanks!

**Pull request merged**